src/signer/src/main/java/ee/ria/xroad/signer/protocol/handler/GetMemberSigningInfoRequestHandler.java

```diff
@@ -68,7 +68,8 @@ protected Object handle(GetMemberSigningInfo message) throws Exception {
 
         if (memberKeys.isEmpty()) {
             throw CodedException.tr(X_UNKNOWN_MEMBER, "member_certs_not_found",
-                    "Could not find any certificates for member '%s'", message.getMemberId());
+                    "Could not find any certificates for member '%s'. "
+                        + "Are you sure tokens containing the certifications are logged in?", message.getMemberId());
```

Comment:
Making modifications to texts shown in UI by changing signer code, which is not aware of UI, and has also other non-UI users, is a bit suspect. This exception is also thrown at least to `xroad-proxy`, `signer-cli`, and in theory could by unwanted in those. It is just a text change so the probability of it breaking things should be minimal, though.

Maybe we should add a way to define core error messages (or overrides to default messages) inside proxy-ui-api module. 

Let's work on this if we get more of these UI-rooted needs to change error messages. Now next release is quite soon, there's no sense in opening this issue more at this point.